protobuf.bzl

```diff
@@ -193,6 +193,30 @@ def cc_proto_library(
       **kargs)
 
 
+def internal_gen_well_known_protos_java(srcs):
+  """Bazel rule to generate the gen_well_known_protos_java genrule
+
+  Args:
+    srcs: the well known protos
+  """
+  root = Label("%s//protobuf_java" % (REPOSITORY_NAME)).workspace_root
```

Comment:
Hmm, never-mind, it probably doesn't matter that much as it will be only used internally. Can you double check if this works for the following:
1) the standalone protobuf package
2) when it's imported as a repository and bind to external.
3) when it's imported as a repository and used directly

At least we need to make (1) and (2) work.